### PR TITLE
Removal of in repo docs - doc now in project-2a-docs repo #367

### PR DESCRIPTION
**Pull request by p5ntangle**
Removal of in repo docs, docs previously moved to https://github.com/Mirantis/project-2a-docs

#367

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
-# Mirantis Hybrid Cloud Platform
+# Mirantis Hybrid Multi Cluster
+
+## Overview
+
+Mirantis Hybribd Multi Cluster is part of Mirantis Project 2A which is focused on delivering a
```
Comment (a13x5) [inline README.md:5]: `Hybrid`

File: README.md
Comment (a13x5) [inline README.md:1]: All links in README.md should be fixed(pointed to new repo). For example this `[AWS](docs/aws/main.md#prepare-the-aws-infra-provider)` will return 404, which isn't nice.

File: docs/dev.md
Comment (a13x5) [inline docs/dev.md:1]: I would suggest to leave this one in place, since it's highly related to make targets and other dev related info.

**Review by a13x5 — CHANGES_REQUESTED** [review]
Several notes regarding links and files deleted.

File: README.md (hunk 1 above)
Comment (p5ntangle) [reply README.md:5]: Good catch, thats what I get for multi-threading

File: README.md
Comment (p5ntangle) [reply README.md:1]: I will actually remove the links, they will be to complicated to maintain long term, I will just put a reference to the docs repo as a whole.

File: docs/dev.md
Comment (p5ntangle) [reply docs/dev.md:1]: Fair point, I will add it back in, its going to need a bunch of fixes as there are a whole lot of broken links in it anyway

**Review by a13x5 — APPROVED** [review]
LGTM

**Pull request merged**